pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -26,15 +27,20 @@ let
     inherit writeShellScript nix-prefetch-git curl jq xe src;
   };
 
+  fetch = (v: fetchgit {inherit (v) url rev sha256 fetchSubmodules; });
+
   grammars =
-    let fetch =
-      (v: fetchgit {inherit (v) url rev sha256 fetchSubmodules; });
-    in runCommand "grammars" {} (''
+    runCommand "grammars" {} (''
        mkdir $out
      '' + (lib.concatStrings (lib.mapAttrsToList
             (name: grammar: "ln -s ${fetch grammar} $out/${name}\n")
             (import ./grammars))));
 
+  builtGrammars = let
+    change = name: grammar:
+      callPackage ./library.nix { language = name; version = "placeholder"; source = fetch grammar;}
```

Comment:
Let’s use the tree-sitter version here

Reply:
It’s still `"placeholder"` as far as I can see

Reply:
sorry I messed up the commit. Fixed.